pkg/stanza/operator/config.go

```diff
@@ -16,6 +17,11 @@ type Config struct {
 	Builder
 }
 
+type BuildInfoInternal struct {
```

Comment:
Depending on how many individual parts of `CreateSettings` we need, I may be preferable to pass in each parameter separately, rather than have a wrapper like this.

Reply:
I agree to only pass the parts of CreateSettings that we need, but I think a wrapper is also needed. I raised some questions in the comment below.